src/utils/typeguards.ts

```diff
@@ -153,13 +152,6 @@ export function isNumberedPortfolio(value: unknown): value is NumberedPortfolio
   return value instanceof NumberedPortfolio;
 }
 
-/**
- * Return whether value is an Asset
- */
-export function isAsset(value: unknown): value is Asset {
-  return value instanceof Asset;
-}
-
```

Comment:
Should we provide TypeGuards for both kind of assets?

Reply:
Yeah, thats probably a good idea